pkg/defaults/policies/files/root_user.json

```diff
@@ -12,6 +12,267 @@
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment machine-approver in namespace openshift-cluster-machine-approver",
+      "deployment": {
+        "name": "machine-approver",
+        "scope": {
+          "namespace": "openshift-cluster-machine-approver"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment kube-apiserver-operator in namespace openshift-kube-apiserver-operator",
+      "deployment": {
+        "name": "kube-apiserver-operator",
+        "scope": {
+          "namespace": "openshift-kube-apiserver-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment ingress-canary in namespace openshift-ingress-canary",
+      "deployment": {
+        "name": "ingress-canary",
+        "scope": {
+          "namespace": "openshift-ingress-canary"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment ingress-operator in namespace openshift-ingress-operator",
+      "deployment": {
+        "name": "ingress-operator",
+        "scope": {
+          "namespace": "openshift-ingress-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment cluster-cloud-controller-manager-operator in namespace openshift-cloud-controller-manager-operator",
+      "deployment": {
+        "name": "cluster-cloud-controller-manager-operator",
+        "scope": {
+          "namespace": "openshift-cloud-controller-manager-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment blackbox-exporter in namespace openshift-route-monitor-operator",
+      "deployment": {
+        "name": "blackbox-exporter",
+        "scope": {
+          "namespace": "openshift-route-monitor-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment thanos-ruler-user-workload in namespace openshift-user-workload-monitoring",
+      "deployment": {
+        "name": "thanos-ruler-user-workload",
+        "scope": {
+          "namespace": "openshift-user-workload-monitoring"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment openshift-apiserver-operator in namespace openshift-apiserver-operator",
+      "deployment": {
+        "name": "openshift-apiserver-operator",
+        "scope": {
+          "namespace": "openshift-apiserver-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment etcd-operator in namespace openshift-etcd-operator",
+      "deployment": {
+        "name": "etcd-operator",
+        "scope": {
+          "namespace": "openshift-etcd-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment openshift-config-operator in namespace openshift-config-operator",
+      "deployment": {
+        "name": "openshift-config-operator",
+        "scope": {
+          "namespace": "openshift-config-operator"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment m in namespace i",
```

Comment:
Really? m and i are the names?

Reply:
I wrote a script to generate jsons.  This must be when there was some bug in the script. fixed it now